### DISCUSSION TITLE
Is there a way to alter the color in every frame to a different color?

### DISCUSSION THREAD

**Discussion by ll-ghin** [keiwando/toonsquid#623] [Q&A] [answered]
For example, suppose I have a drawing in 60 individual frames across an animation and the drawing is a black vector line that is moving.

Is there a way I could change every line in the frame to a green line without using the bucket tool and filling manually for every frame?  Thanks!

**Comment by kavlor** [comment]
You alter vector colours on the right side of the screen by clicking the '=' symbol above the trash can symbol and the pull out menu has options for altering fill and stroke under Path Style. As long as the frames are tweened you can alter every frame by setting key frames either end to the colour you want by copying and pasting the keyframe from the first to the last.

**Reply by ll-ghin** [reply]
Sorry, the original post is poor wording by me.  It is not a line draw via the path tool (I used a line as an example), but instead a collection of frames of stick figures all drawn with the vector brush tool.

**Reply by FoxeyeStudios** [reply, marked as answer]
If you’re not using the path tool, then your vector lines will remain the same color throughout the animation. You can however use a clipping mask to change its color. You need to make a separate track and make a rectangle shape path that fills up the entire frame, make sure to have that track over your stick figure. Then, have the path track “toggle mask” and you have your figure a different color. Best part about this method is you can use key frames to change the color mid animation by using the same rectangle path you made, so you don’t have to be making new shapes to change colors if that’s what you want.

**Reply by kavlor** [reply]
Depending on usage scenario you could also use Blend mode. Similar, using a large rectangle. This would recolour everything below it not just one layer but it could be great where you actually want to do this and make it translucent if you just want to tint the scene and even mess with the gradient fill.
